File: C/Lista_01(Estrutura_Sequencial)/Duracao.c
#include <stdio.h>

int main () {

    int duracao;

    printf("Digite a duracao em segundos: ");
    scanf("%d", &duracao);

    int horas = duracao/3600;
    int minutos = (duracao%3600)/60;
    int segundos = (duracao%3600)%60;

    printf("%d:%d:%d", horas, minutos, segundos);
    
    return 0;
    
}

File: Java/Lista_01(Estrutura_Sequencial)/Duracao.java
import java.util.Scanner;

public class Duracao {

	public static void main(String[] args) {
		
		Scanner s = new Scanner(System.in);
		
		System.out.print("Digite a duracao em segundos: ");
		int duracao = s.nextInt();
		
		int horas = duracao/3600;
		
		int minutos = duracao%3600/60;
		
		int segundos = duracao%3600%60;
		
		System.out.println(horas + ":" + minutos + ":" + segundos);
		
		s.close();
		
	}

}
